test/bin/eshost.js

```diff
@@ -26,6 +26,8 @@ if (isWindows) {
   });
 }
 
+// To avoid messing with a real configuration, use a test-local config
+Config.defaultConfigPath = path.resolve('./test/config/.eshost-config.json');
```

Comment:
Is this necessary? Seems like the --config option takes care of this.

Another option is to use an environment variable to control where the defaultConfigPath is. Then you don't need to pass it in.